test/onnx/test_utility_funs.py

```diff
@@ -309,6 +309,27 @@ def forward(self, x):
             assert node.kind() != "onnx::Sqrt"
         assert len(list(graph.nodes())) == 1
 
+    def test_constant_fold_shape(self):
+        class ShapeModule(torch.nn.Module):
+            def __init__(self):
+                super(ShapeModule, self).__init__()
+                self.register_buffer("weight", torch.ones(5))
+
+            def forward(self, x):
+                shape = torch.tensor(list(self.weight.shape))
```

Comment:
nit: list() is not necessary here. Could be: shape = torch.tensor(self.weight.shape)